src/lib/npm.ts

```diff
@@ -38,20 +39,50 @@ const getNpmPath = async () => {
   return join(node_modules(), ".bin", "npm");
 };
 
+async function runNpm(args: Omit<ExecPromiseArg, "command">) {
+  let res: string;
+
+  try {
+    res = await execPromise({
```

Comment:
maybe first check that npmPath exists and is an executable?

and then use the correct path. could even cache it.